localization/english/from_castles_to_bastilles_l_english.yml

```diff
@@ -111,4 +111,14 @@
   form_british_raj_decision_tooltip: "Consolidate the Indian Empires into the British Raj"
   form_british_raj_decision_effect_tooltip: "All Indian Empires assimilate under British Raj and you gain the British Raj Empire title"
   form_british_raj_decision_confirm: "The Jewel is mine!"
-  e_british_raj: "British Raj"
\ No newline at end of file
+  e_british_raj: "British Raj"
+
+  #German Culture Decision
+  form_german_culture_decision: "Form Unified German Culture"
+  form_german_culture_decision_desc: "The time has come for us to unify our cultures into one"
+  form_german_culture_decision_tooltip: "Consolidate all Germanic people into one German culture"
+  form_german_culture_decision_effect_tooltip: "All Germanic people assimilate under one Unified German Culture"
+  form_german_culture_decision_confirm: "We are One!"
+  unified_german: "Unified German Culture"
+  german_unified_lands: "Unified German Land"
+  get_all_innovations_from: "Get All Innovations From"
```

Comment:
Why add localization for `get_all_innovations_from`?
Localizations should only be added if we are expecting this to appear to the player

Reply:
Removed that part in the localizations